crates/ruma-events/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ Improvements:
 - Stabilize support for muting in VoIP calls, according to Matrix 1.11
 - All the root `Any*EventContent` types now have a `EventContentFromType` implementations
   automatically derived by the `event_enum!` macro.
+- `CallMemberEventContent` now supports two different formats: Session memberships and Legacy memberships.
+The new format (Session) is required to reliably display the call member count (reliable call member events).
+`CallMemberEventContent` is now an enum to model the two different formats.
+- `CallMemberStateKey` (instead of `String`) is now used as the state key type for `CallMemberEventContent`.
```

Comment:
The state key was an `OwnedUserId`, not a `String`.

Reply:
It was a `OwnedUserId` first than a `String` since we needed to support both `Legacy` and `Session` and now it is its own custom type.
But I guess there was no release when it was a `String` so for the release notes we want to hide the `String` transition.